src/esp/bindings_js/bindings_js.cpp

```diff
@@ -79,9 +82,16 @@ vec4f eulerToQuaternion(const vec3f& q) {
       .coeffs();
 }
 
+vec4f quaternionMultiply(const vec4f& q1, const vec4f& q2) {
+  auto product = Magnum::Quaternion(quatf(q1)) * Magnum::Quaternion(quatf(q2));
+  return vec4f(product.data()[0], product.data()[1], product.data()[2],
+               product.data()[3]);
```

Comment:
`vec4f` is Eigen, right?

```suggestion
  return Mn::EigenIntegration::cast<vec4f>(product);
``` 

should work. [Unfortunately explicit cast doesn't work in this direction because Eigen is special.](https://doc.magnum.graphics/magnum/integration_2src_2Magnum_2EigenIntegration_2Integration_8h.html#EigenIntegration-math)